Instacard/models/SelectedImageModel.swift

```diff
@@ -27,24 +27,19 @@ class SelectedImageModel: ObservableObject {
     
     struct SelectedImage: Transferable {
         let image: Image
+        let cgImage: CGImage
         
         static var transferRepresentation: some TransferRepresentation {
             DataRepresentation(importedContentType: .image) { data in
-            #if canImport(AppKit)
-                guard let nsImage = NSImage(data: data) else {
-                    throw TransferError.importFailed
-                }
-                let image = Image(nsImage: nsImage)
-                return SelectedImage(image: image)
```

Comment:
Removing this AppKit code which effectively removes support for Mac OS, which we weren't supporting anyway. It was hard to figure out how to go from NSImage to CGImage, and with this deleted, it doesn't matter :)